File: 프로그래머스/unrated/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.js
function solution(strArr) {
    var answer = [];
    
    for(var i=0; i<strArr.length; i++){
        if(i%2 == 0){
            answer.push(strArr[i].toLowerCase());
        }else{
            answer.push(strArr[i].toUpperCase());
        }
    }
    return answer;
}

File: 프로그래머스/unrated/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.java
class Solution {
    public String[] solution(String[] strArr) {
        
        //삼항연산자로 풀이
        for(int i=0; i<strArr.length; i++){
          
             strArr[i] = i%2 == 0? strArr[i].toLowerCase() : strArr[i].toUpperCase();
        }
        return strArr;
    }
}

//class Solution {
//    public String[] solution(String[] strArr) {
        
//        for(int i=0; i<strArr.length; i++){
//            if(i%2 == 0){
//                strArr[i] = strArr[i].toLowerCase();
//            }else{
//                strArr[i] = strArr[i].toUpperCase();
//            }
//        }
//        return strArr;
//    }
//}